pages/1.12/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -93,7 +93,8 @@ This page contains the configuration parameters for both DC/OS Enterprise and DC
 | ca_certificate_path                   | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/1.12/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
 | ca_certificate_key_path           | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/1.12/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
 | ca_certificate_chain_path       | Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/1.12/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. [enterprise type="inline" size="small" /] |
-| [security](#security-enterprise)                               | The security mode: disabled, permissive, or strict. [enterprise type="inline" size="small" /] |
+| [iam_ldap_sync_interval](/1.12/security/ent/ldap/ldap-sync/) | Interval in seconds between ldap sync operations. [enterprise type="inline" size="small" /] | 
+|[security](#security-enterprise)                               | The security mode: disabled, permissive, or strict. [enterprise type="inline" size="small" /] |
```

Comment:
While we're here we should remove `disabled` security mode. We don't support that mode in 1.12 and going forward. CC @jongiddy

Reply:
Hi @jongiddy, I scanned through and changed where it looked like the work had been reverted, if you find anything after I've merged this branch in please ping me or just make me a ticket :)